packages/gateway/src/features/user/database.ts

```diff
@@ -30,7 +30,7 @@ export async function stop() {
 }
 
 export async function start() {
-  logger.info('REDIS_HOST', REDIS_HOST)
+  logger.info('REDIS_HOST, ' + JSON.stringify(REDIS_HOST))
```

Comment:
```suggestion
  logger.info(`REDIS_HOST, ${JSON.stringify(REDIS_HOST)}`)
```
Let's try to use template literals where possible, just to make the code a bit cleaner.